### ISSUE TITLE
Verified Facet Broken

### ISSUE THREAD

**Issue by garyluu** [dockstore/dockstore#2376]
### Expected behaviour
Verified facet to show the right amount of verified tools and workflows

Ideally, the index should be fixed so that the UI does not need to do crazy workarounds in order to get the correct number.

### Actual behaviour
The number shows 16 but there are in fact 28 tools and workflows

### Steps to reproduce the behaviour 
- Go to https://staging.dockstore.org/search?searchMode=files
Note that the verified checkbox number says 16
- Click on it
Now the number says 28

┆Issue is synchronized with this [Jira Bug](https://ucsc-cgl.atlassian.net/browse/DOCK-673)
┆Issue Type: Bug
┆Fix Versions: Dockstore 1.7
┆Sprint: Seabright Sprint 18 Skipjack
┆Issue Number: DOCK-673
┆Epic: Fix Facets

**Comment by cricketsloan** [comment]
➤ Natalie Perez commented:

may not go to 1.6.x

**Comment by denis-yuen** [comment]
Looks fixed with 
```
API - 1.7.0-rc.1 
UI - 2.4.0-rc.1
``` 
but might have introduced an issue, the verified source looks like an array 

![Screenshot from 2019-09-11 13-34-39](https://user-images.githubusercontent.com/1730679/64720676-0d50e180-d499-11e9-997e-559d2a884725.png)

**Comment by denis-yuen** [comment]
Note that production also has a verified source for workflows of `GA4GH/DREAM team` that seems missing in staging

**Comment by garyluu** [comment]
ya, the verified source is going to be an issue

**Comment by denis-yuen** [comment]
As discussed, don't really have a preference whether this is addressed in this issue or a follow-up issue if it looks too unrelated. But it would be nice to take a look for 1.7.0, possibly the code that converts our models and bulk uploads to elastic search can be improved

**Comment by denis-yuen** [comment]
Think this is waiting on #2887

**Comment by denis-yuen** [comment]
Counts are correct on staging
Verified source is missing, demoting source to non-critical 1.7.0

**Comment by denis-yuen** [comment]
Workaround found, need to reindex again after start?
